causalpy/pymc_models.py

```diff
@@ -113,3 +115,30 @@ def build_model(self, X, y, coords):
             sigma = pm.HalfNormal("sigma", 1)
             mu = pm.Deterministic("mu", pm.math.dot(X, beta), dims="obs_ind")
             pm.Normal("y_hat", mu, sigma, observed=y, dims="obs_ind")
+
+
+class BARTModel(ModelBuilder):
+    "Class for building BART based models for meta-learners."
+
+    def __init__(self, sample_kwargs=None, m=20, sigma=1):
+        self.m = m
+        self.sigma = sigma
+        super().__init__(sample_kwargs)
+
+    def build_model(self, X, y, coords=None):
+        with self:
+            self.add_coords(coords)
+            X = pm.MutableData("X", X, dims=["obs_ind", "coeffs"])
```

Comment:
I think is better to use another variable name for the Data object (this has been a problem for me in the past). Something like 
```python
 X_ = pm.MutableData("X", X, dims=["obs_ind", "coeffs"])
 ```